### ISSUE TITLE
update for navi reset patch

### ISSUE THREAD

**Issue by rkitover** [feniksa/gentoo_ACS_override_patch#2]
Apparently you just need to replace `ioremap_nocache()` with `ioremap()`, and put the table entries on a different line, here is my patch for 5.8:

https://gist.github.com/rkitover/6624d9bab13e7738bd92e5a7ff2fbb67

**Comment by feniksa** [comment]
Hi @rkitover ,

Thanks, I will move your patch into this repo. Will make in few days

**Comment by feniksa** [comment]
@rkitover this is still actual for navi?

I have limited access to vcard with navi, can't check

**Comment by rkitover** [comment]
Yeah I've used this with my 5700xts and qemu vms, still applies to all recent kernels afaik.
